### PR TITLE
feat(Tooltip): add a max width prop

### PR DESCRIPTION
**Pull request by talkor**
https://monday.monday.com/boards/3532714909/pulses/5570948205

### CODE REVIEW

File: src/components/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -115,6 +115,11 @@ interface TooltipBaseProps extends VibeComponentProps {
    * Overwrites z-index of the tooltip
    */
   zIndex?: number;
+  // TODO: make defualt next major
+  /**
+   * Limit tooltip to 240px
+   */
+  maxWidth?: boolean;
```
Comment (talkor) [inline src/components/Tooltip/Tooltip.tsx:122]: Couldn't find a better name that will be short enough, open to suggestions

File: src/components/Tooltip/__stories__/tooltip.stories.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ const tooltipTemplate = args => {
         // The modifier's purpose is to prevent the tipseen from being displayed when the user scrolls the story upwards / downwards.
         // Therefore, there is no need to move this prop in your implementations.
         modifiers={modifiers}
+        maxWidth
```
Comment (talkor) [inline src/components/Tooltip/__stories__/tooltip.stories.js:22]: Internally showing it in the example so users will take this code as an example

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (shlomitc) [reply src/components/Tooltip/Tooltip.tsx:122]: setOfficialMaxWidth

File: src/components/Tooltip/Tooltip.module.scss [hunk 3]
```diff
@@ -11,6 +11,10 @@ $tooltip-z-index: 9999999999999999;
   @include font-input();
   max-width: var(--tooltip-max-width, 50vw);
   word-break: break-word;
+
+  &.maxWidth {
```
Comment (YossiSaadi) [inline src/components/Tooltip/Tooltip.module.scss:15]: I would consider maybe a naming change, to make it clearer

File: src/components/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (YossiSaadi) [reply src/components/Tooltip/Tooltip.tsx:122]: Even `withMaxWidth` or something similar, `limitWidth`

**Review by YossiSaadi — APPROVED** [review]
Looks cool, but I would indeed consider a naming change

**Pull request merged**